src/js/components/molecules/tabs/tabs.ts

```diff
@@ -29,23 +29,21 @@ export class Tabs extends Vue implements TabsProps {
     setActiveTab() {
         const tabs = this.$refs.tab;
 
-        if (!tabs) { return; }
+        if (!tabs) return;
```

Comment:
avoid these single line if statements. @see https://www.crockford.com/code.html#if%20statement

```suggestion
        if (!tabs) {
            return;
        }
```

Reply:
Do we actively follow these guidelines? Because I personally don't like using curly braces when there's no `else`, I would much rather have `if(!tabs) return` or (even better) `!tabs && return`. That's way quicker and doesn't consume as much space

Reply:
we have been using this style for at least 6 months and it's an industry standard.